### DISCUSSION TITLE
Accept 2-letter locales in localizedAttributes

### DISCUSSION THREAD

**Discussion by tacman** [meilisearch/product#779] [Feedback & Feature Proposal]
From https://github.com/meilisearch/meilisearch/discussions/4904#discussioncomment-10590331
```json
{
  "localizedAttributes": [
      {"locales": ["es"], "attributePatterns": ["*.es"]},
      {"locales": ["en"], "attributePatterns": ["*.en"]},
      {"locales": [], "attributePatterns": ["*"]}
  ]
}
```

It would be a lot easier if you also supported 2-character ISO codes (https://en.wikipedia.org/wiki/List_of_ISO_639_language_codes) instead of just the 3-character.

The reason is that many web developers code around the request locale, and using the 3-letter code isn't very standard.  So I find myself trying to find a library that makes the mapping.

That is, I'd like

```json
{
  "localizedAttributes": [
      {"locales": ["es"], "attributePatterns": ["*_es"]},
      {"locales": ["en"], "attributePatterns": ["*_en"]},
      {"locales": [], "attributePatterns": ["*"]}
  ]
}
```

**Comment by dureuill** [comment]
Implemented by https://github.com/meilisearch/meilisearch/pull/4937
